base_layer/p2p/Cargo.toml

```diff
@@ -4,5 +4,6 @@ version = "0.0.1"
 edition = "2018"
 
 [dependencies]
-tari_crypto = { path = "../../infrastructure/crypto"}
-tari_comms = { path = "../../infrastructure/comms"}
+tari_crypto = { version = "0.0.1", path = "../../infrastructure/crypto"}
```

Comment:
Are these version number necessary? They will presumably have to be updated when versions are bumped

Reply:
It's a bit of a pain to manage, but Cargo complains if we want to publish the crates and they're not there